### ISSUE TITLE
Logo and branding

### ISSUE THREAD

**Issue by pathumego** [reliefsupports/reliefsupports.org-v1#147]
Following up on @danishka  [request](https://twitter.com/danishkanavin/status/869501789812342784) for logo. 
A simple brief and ideas for the logo;
- community
- strength 
- collective effort
- relief support
- technology

**Comment by pathumego** [comment]
[edited] PLEASE VOTE TO LOGOS USING THE THUMBS UP REACTION IN THE SEPERATE COMMENTS POSTED BELOW.

This is a proposal for the logo based on the ideas. These are drafts to select an icon. If we select an icon from this, we will work on different layout options and usages such as the android and iOS app icons, website header logo, social media icons. 

<img width="1412" alt="screen shot 2017-05-31 at 12 49 52 pm" src="https://cloud.githubusercontent.com/assets/6846609/26620261/b289f112-45ff-11e7-82a4-bb4ab4b24057.png">

**Comment by kasunben** [comment]
;
@naveendw @chamilad  @gayanhewa

**Comment by pathumego** [comment]
and it should be `reliefsupports` not `reliefsupport` :) Ill fix that if we select any of these.

**Comment by thilinag** [comment]
@pathumego The text is going to be on the side in header logo ?

**Comment by danishka** [comment]
@isuru4lk ^

**Comment by pathumego** [comment]
@thilinag Yes, consider this a draft, Ill make options and layouts once we select an Icon

**Comment by yoosuf** [comment]
I love the third form top.

**Comment by thilinag** [comment]
put logos in separate comments and count thumbs up ?

**Comment by pathumego** [comment]
![relief support logo-02](https://cloud.githubusercontent.com/assets/6846609/26619011/3e1fd09e-45fa-11e7-8cba-09bd91c91509.png)

**Comment by pathumego** [comment]
![relief support logo-05](https://cloud.githubusercontent.com/assets/6846609/26619010/3e1cb49a-45fa-11e7-9862-268fdba0b136.png)

**Comment by pathumego** [comment]
![relief support logo-03](https://cloud.githubusercontent.com/assets/6846609/26619014/3e4ade92-45fa-11e7-83d6-2e1426d51ee0.png)

**Comment by pathumego** [comment]
![relief support logo-04](https://cloud.githubusercontent.com/assets/6846609/26619013/3e4931be-45fa-11e7-9148-9db6d27df9d7.png)

**Comment by pathumego** [comment]
<img width="711" alt="relief support logo-01" src="https://cloud.githubusercontent.com/assets/6846609/26619012/3e48e254-45fa-11e7-8069-c5eac912d0e3.png">

**Comment by lishwebber** [comment]
![logo-relief-supports](https://cloud.githubusercontent.com/assets/29089374/26621859/e17d5382-4605-11e7-80be-15d35aa363ee.jpg)

**Comment by thinkholic** [comment]
1st one is neat. :)

**Comment by pathumego** [comment]
@thinkholic there is a new proposal in the works with the FB and promo team.

**Comment by harsha935** [comment]
What about this
![relief_support](https://cloud.githubusercontent.com/assets/6870173/26626818/11a26d76-4616-11e7-8c03-8f5d1fb0043c.png)

Already made an app icon for android application
![android_icon](https://cloud.githubusercontent.com/assets/6870173/26626925/57c0c6f4-4616-11e7-9002-424a7965386f.png)

**Comment by pathumego** [comment]
Following up on the brief more directly relating to floods and current situation. Didn't add colors yet
<img width="807" alt="screen shot 2017-05-31 at 5 20 42 pm" src="https://cloud.githubusercontent.com/assets/6846609/26634358/03979a30-4634-11e7-9257-141e1f839409.png">

**Comment by danishka** [comment]
Subash Shaminda Kuruwitage contributed with these designs 
[OPTION-1.pdf](https://github.com/reliefsupports/reliefsupports.org/files/1043324/OPTION-1.pdf)
[OPTION-2.pdf](https://github.com/reliefsupports/reliefsupports.org/files/1043351/OPTION-2.pdf)

**Comment by danishka** [comment]
Please add all submits here as we should give them required recognition.

**Comment by chamilad** [comment]
IMO, our branding shouldn't only be about this flooding. It should cover the idea that Relief Supports applies to any disaster-related situation. WDYT?

**Comment by harsha935** [comment]
@chamilad exactly!

**Comment by kosalaya** [comment]
I made some changes to the logo that follows the brief with the direct message on the current situation. However, I do agree with @chamilad here. I think it's better we opt for a logo that reflects the long-term objective of the project. I am also posting one I designed earlier to the brief posted on fb by @danishka. up to you guys to select one :) Cheers!

<img width="675" alt="screen shot 2017-06-01 at 11 48 59 am" src="https://cloud.githubusercontent.com/assets/8073976/26667220/c0735ff8-46c1-11e7-83a7-add7287d52b8.png">
<img width="667" alt="screen shot 2017-06-01 at 11 48 44 am" src="https://cloud.githubusercontent.com/assets/8073976/26667231/c6e2d152-46c1-11e7-89ce-87bb853b96ca.png">

**Comment by kasunben** [comment]
@chamilad 👍 exactly that is the point.

**Comment by danishka** [comment]
1
![rs-white](https://user-images.githubusercontent.com/178024/26859575-3aa40ab8-4b6b-11e7-9ded-c55a6aaa1b31.png)
2
![rs-green](https://user-images.githubusercontent.com/178024/26859580-3e0455c8-4b6b-11e7-8d68-5f0c2b8599bd.png)
3
![rs-gray](https://user-images.githubusercontent.com/178024/26859581-412aafcc-4b6b-11e7-8efb-a8c98467f6a1.png)
4
![rs-gray-green](https://user-images.githubusercontent.com/178024/26859582-43bd2e72-4b6b-11e7-8ca3-60df4f485ca9.png)
5
![screen-shot-2017-06-06-at-8 15 52-pm](https://user-images.githubusercontent.com/178024/26859583-45b23a24-4b6b-11e7-9541-23fd2a67eb09.jpg)

**Comment by kasunben** [comment]
Vote for 1 n 3.

**Comment by ravithb** [comment]
4

**Comment by kpgtharaka** [comment]
4

> brief and ideas for the logo;
> 
> community
> strength
> collective effort
> relief support
> technology

other than this we can say quick support(speed)

**Comment by callkalpa** [comment]
Voting for 4

**Comment by harsha935** [comment]
Voting for 4

**Comment by chameeracd** [comment]
4

**Comment by Ruwatech** [comment]
4 i guess
